db/migrate/20230404210918_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Also here, Kindly write the query like this so the foreign key `author_id` will add relationships between user and like tables :


```ruby
  add_reference :likes, :author, null: false, foreign_key: { to_table: :users }
```

Reply:
Here is the ERD diagram ⬇️ 

![image](https://user-images.githubusercontent.com/79058364/229947668-c6c87e41-548d-4da3-9e65-e465e0d7bd5a.png)

Reply:
<img width="1417" alt="Captura de pantalla 2023-04-04 a la(s) 18 29 25" src="https://user-images.githubusercontent.com/99061965/229951472-fad5a99d-3c9d-4cd3-99ca-a7de12c9bbfd.png">

Reply:
### Hello @PierreBarba, sorry of this inconvient, Kindly request you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).